src/main/clojure/clara/rules/compiler.clj

```diff
@@ -31,7 +31,8 @@
             Accumulator
             ISystemFact]
            [java.beans
-            PropertyDescriptor]
+            PropertyDescriptor
+            IndexedPropertyDescriptor]
```

Comment:
This new import does not seem like it is needed.

Reply:
cleaned up:
https://github.com/cerner/clara-rules/pull/449/commits/27e120f65dbaa595505e0f858c31921af9124e86